modules/thermal_hydraulics/doc/content/source/ics/SumIC.md

```diff
@@ -1,20 +1,9 @@
 # SumIC
 
-!alert construction title=Undocumented Class
-The SumIC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /ICs/SumIC
 
-## Overview
-
-!! Replace these lines with information regarding the SumIC object.
-
-## Example Input File Syntax
-
-!! Describe and include an example of how to use the SumIC object.
+The value of the other variables in the sum is naturally equal to their initial conditions,
+as initial conditions are only run once at the beginning of the simulation.
```

Comment:
I don't understand what point you're making / why you're clarifying this.

Reply:
just that things are as you'd want eg, this IC is ran last. Maybe it s not needed